internal/app/repository/file.go

```diff
@@ -0,0 +1,98 @@
+package repository
+
+import (
+	"encoding/json"
+	"errors"
+	"io"
+	"log"
+	"os"
+)
+
+//implements Repository interface
+type FileRepository struct {
+	fileStoragePath string
+}
+
+type fileRecord struct {
+	ID          int    `json:"id"`
+	OriginalURL string `json:"original_url"`
+}
+
+func openFile(fileName string, flag int) (*os.File, error) {
+	file, err := os.OpenFile(fileName, flag, 0777)
+	if err != nil {
+		return nil, err
+	}
+	return file, nil
```

Comment:
Можно так) Есть вообще вопрос а стоит ли os.OpenFile в отдельную функцию оборачивать. 
```suggestion
	return os.OpenFile(fileName, flag, 0777)
```

Reply:
Удалил общую функцию, похоже действительно в ней нет смысла.